00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,593 @@
+and should therefore be done first. The exact process of triage is detailed
+separately in the [Triage](#triage) section. Triage should not be strictly
+time-bound; if productive discussions are happening around a new issue or PR,
+those should be allowed to happen naturally.
+
+The decision to make triage a part of daily standup is deliberate: it allows
+issues and PRs filed by "drive-by" contributors (that is, people not part of
+the project that still wish to contribute) to receive human consideration after
+a maximum of 24 working hours -- a desirable property for open source.
+
+Following triage, the scrum master will pull up the sprint board for an informal
+board review. This does not need to go over every single issue; rather, it
+should be an opportunity for team members to point out highlights or interesting
+updates to their tasks, and request help on blockers or otherwise. To facilitate
+this, the scrum master should ask the meeting's attendees for a quick show of
+hands on if they have something they wish to talk about; this will allow the
+scrum master to keep time constraints in mind and move conversations along if
+they're taking up too much time.
+
+Finally, the rest of the meeting should be dedicated to open-ended project
+discussion. If there are guests to the call, the team should welcome them and
+allow them to ask questions. If there are any other topics to cover, they should
+be brought up here.
+
+Standup should take no longer than a half-hour, and should be scheduled as
+follows:
+
+- 10-15 min: triage
+- 15-20 min: Board review/blockers/project discussion (however long is left
+  after triage)
+
+### Review
+
+The final meeting is **Review**, which represents the end of a
+sprint. It is a time for the whole team to reflect on work accomplished, demo
+new things, and discuss how to approach tasks that didn't get completed.
+
+Most of these topics will take the form of discussion. The only significant
+process to detail here is **Retrospective**, the workflow for reviewing issues
+and PRs from the last sprint.
+
+![Sprint Retro diagram](./sprint-retro.svg)
+
+Any issues that don't have an associated PR are assumed to be "not started".
+However, any issue that made it into the sprint in the first place is probably
+still important, so it should be considered as such for the next sprint as well.
+Thus, it will be sent to the next Planning meeting as a Proposed issue (see
+[Planning](#planning) for more details).
+
+If it _does_ have an associated PR, the next question to ask is whether that
+associated PR was merged or not. If it was, then the issue is completed. If not,
+that means the PR is still in review and likely just needs a bit more review
+time. Thus, it should stay as-is in the Under Review column.
+
+Retrospective should happen immediately following demos (if there are any).
+Putting demos first allows those interested to see them without having to sit
+through the minutiae of project discussion.
```

Comment:
This is good and makes sense, but makes even more sense if you consider/make explicit that all stakeholders should be invited to the demo: some of the stakeholders have no interest in the minutiae of project discussion but are very interested in seeing the progress made. We start with the demo to let them leave after.

Suggestion, add somewhere above the following:

> All stakeholders are invited to the demo, so they can get a sense of the progress made in the last sprint, or since they were last able to attend a demo.

Reply:
Added something to this effect.